windows/gui-source/Kolibri/Kolibri.cpp

```diff
@@ -44,6 +45,31 @@ wchar_t * getStr(int strId) {
 	return finalStr;
 }
 
+char * getKolibriLinkAddress() {
+	/*
+	This will return the current Kolibri running link url.
+	It will fetch the PORT value from server.pid file.
+	*/
+	char * kolibriHomeEnv = getenv("KOLIBRI_HOME");
+	if (kolibriHomeEnv == NULL) {
+		kolibriHomeEnv = "";

```

Comment:
`KOLIBRI_HOME` is not defined we should use `GetEnv('HOMEPATH') + '\.kolibri'`.

If you do

```cpp
char * pidFile = joinChr(kolibriHomeEnv, configName);
```

and 

```cpp
kolibriHomeEnv = ""
```

then things will break, no?

Reply:
> when KOLIBRI_HOME is not defined we should use GetEnv('HOMEPATH') + '\.kolibri'

Okay @indirectlylit , I'll add it. 

> If you do

> char * pidFile = joinChr(kolibriHomeEnv, configName);
> and

> kolibriHomeEnv = ""
> then things will break, no?

The reason I do that, it's because if the `pidFile` is undefined it will show an error. setting it default to GetEnv('HOMEPATH') will fix this.

Reply:
I think this is related to [the conversation above](https://github.com/learningequality/kolibri-installer-windows/pull/80#discussion_r215186095).

The key idea is that the application should know that the default Kolibri home folder is 

```
GetEnv('HOMEPATH') + '.kolibri'
```

That way, when `KOLIBRI_HOME` is not defined, everything still functions as expected. Furthermore, this should be the common case: `KOLIBRI_HOME` should rarely be defined unless the user is doing something fancy.